runners/genlayer-py-std/src/genlayer/gl/eq_principle.py

```
@@ -20,13 +17,13 @@
 
 
 @_lazy_api
-def eq_principle_strict_eq[T: calldata.Decoded](fn: typing.Callable[[], T]) -> Lazy[T]:
+def strict_eq[T: calldata.Decoded](fn: typing.Callable[[], T]) -> Lazy[T]:
```

Ruff (0.11.9)

`Lazy` may be undefined, or defined from star imports

(F405)